main.py

```diff
@@ -21,6 +31,30 @@ def callback(request):
                 jsonify(message="Currently, the system do not accept a GET request"),
                 405,
             )
-    except IndexError:
+    except:
         return jsonify(message="Something went wrong!"), 400
+
     return "Success"
+
+
+def retry_failed_webhook(transaction_log_service):
+    failed_ivr_logs = transaction_log_service.get_failed_ivr_transaction_log()
+
+    for log in failed_ivr_logs:
+        processed = process_form_data(json.loads(log.payload), log.created_on)
```

Comment:
processing each failed log and passing the actual time that be in the created_on column of the new records that will be created.

Reply:
Is there any reason to pass created_on separately? Isn't it possible to get it from JSON itself?

Reply:
I don't want to make any changes to the payload that's why I am passing it separately.